feat: add additional accessibility features, update all translations

jaredcwhite opened the pull request:
This PR addresses #4749 

- [x] Addresses the issue in full
- [ ] Addresses only certain aspects of the issue

## Description

This adds additional values to `listingFeatures` in `formKeys` so accessibility features can be enumerated on the frontend (they were already possible in the database). In addition, all the translations have been updated (pulls English & Spanish from previous Detroit repo, Filipino/Vietnamese/Chinese are from Google Translate).

## How Can This Be Tested/Reviewed?

Select various accessibility features for a listing in partners, and then verify those features display as expected in public.

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [x] Reviewed in a desktop view
- [ ] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [ ] Added tests covering the changes
- [ ] Made corresponding changes to the documentation
- [x] Ran `yarn generate:client` and/or created a migration when required

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

jaredcwhite commented on sites/public/src/components/listing/ListingViewSeedsHelpers.tsx:
```diff
@@ -119,29 +119,34 @@ export const getHasNonReferralMethods = (listing: Listing) => {
-  const features = Object.keys(listing?.listingFeatures ?? {}).map((feature, index) => {
-    if (listing?.listingFeatures[feature]) {
-      featuresExist = true
+  const enabledFeatures = Object.entries(listing?.listingFeatures ?? {})
```
I refactored these two methods because there was a bug with commas sometimes showing up at the end of a list. This filters out only the values which are true first, then loops through them.

emilyjablonski commented on shared-helpers/src/locales/tl.json:
```diff
@@ -498,24 +498,29 @@
-  "eligibility.accessibility.visual": "Visual",
-  "eligibility.accessibility.wheelchairRamp": "Rampa ng wheelchair",
+  "eligibility.accessibility.acInUnit": "AC sa Unit", 
+  "eligibility.accessibility.accessibleParking": "Accessible Parking Spot", 
```
**note:** I got `Mga Mapupuntahan na Paradahan` from translation `Accessible Parking Spots` here

emilyjablonski reviewed (approved):
✨ beautiful ✨ 
Non-blocking nit bc it's small and bc you have to be at a very specific desktop size, but occasionally the column checkboxes overlap a tiny bit w the text (fine for me to merge over):

![Screenshot 2025-05-12 at 12 29 35 PM](https://github.com/user-attachments/assets/894bf618-39d3-47b6-bcdb-e1e41b6f5aaa)

jaredcwhite commented:
@emilyjablonski Thanks for catching that one translation! I don't know why, but this time I was able to get Google Translation to work as well for the singular version ("Accessible na Paradahan"). So I updated that, and also added some gap between the checkbox columns.
